### PR TITLE
Glossary blog post

### PR DESCRIPTION
**Pull request by brandonhundt**
- This is an SEO play that may be helpful for Dovetail users as well. (Although the [Help doc](https://help.prx.org/hc/en-us/articles/4410588906651-Podcast-Advertising-Sales-Glossary) is more helpful for existing users, this doesn't need to go to the depth of that post)
- Noted Ryan as the author
- Updated the IAB PDFs to 2.1

## To review:

- [ ] `bundle install`
- [ ] `bundle exec jekyll server`
- [ ] Go to `localhost:4000/blog` and click on the post
- [ ] Ensure post is fine

### CODE REVIEW

File: _posts/2022-12-02-dovetail-glossary.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+permalink: /2022/12/02/dovetail-glossary
+title: Dovetail Glossary
+author: Ryan Cavis
+image: /assets/img/hero-features.jpg
```
Comment (cavis) [inline _posts/2022-12-02-dovetail-glossary.md:6]: So you added that `DT - Show Settings Dashboard.png` file, but you're not using it?

(Also wondering if you meant to use a real podcast in that screenshot).

**Review by cavis — APPROVED** [review]
👍 some questions, but looks good to me.

File: _posts/2022-12-02-dovetail-glossary.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+permalink: /2022/12/02/dovetail-glossary
+title: Dovetail Glossary
+author: Ryan Cavis
```
[the remark this answers is not in the document]
Comment (brandonhundt) [reply _posts/2022-12-02-dovetail-glossary.md:5]: You were the principal definer of things here. 🚌

File: _posts/2022-12-02-dovetail-glossary.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+## Ad Serving
+
+**Campaign:** Represents an Inventory Order (IO) for a single advertiser. It can contain many Flights.
+
+**Flight:** One or more creatives plus a set of targeting information to serve with a podcast. Configured with Zones, a date-range, and an Impression goal.
+Ads are fulfilled by reserving Allocations against future Inventory.
+
+**Creative:** Refers to the audio file (MP3, WAV, FLAC) or VAST url containing the ad content.
+
+**VAST:** An acronym that stands for Video Ad Serving Template. VAST is a way for multiple ad servers to communicate and serve ads, replacing individual ad audio files for a campaign with a VAST URL.
```
[the remark this answers is not in the document]
Comment (brandonhundt) [reply _posts/2022-12-02-dovetail-glossary.md:40]: This is how its defined on the help site.

File: _posts/2022-12-02-dovetail-glossary.md (hunk 1 above)
Comment (brandonhundt) [reply _posts/2022-12-02-dovetail-glossary.md:6]: whoops. Will remove.

**Pull request merged**